contracts/src/factory/HyperdriveFactory.sol

```diff
@@ -922,40 +924,53 @@ contract HyperdriveFactory is IHyperdriveFactory {
     }
 
     /// @notice Gets the deployer coordinator at the specified index.
-    /// @param index The index of the deployer coordinator to get.
+    /// @param _index The index of the deployer coordinator to get.
     /// @return The deployer coordinator at the specified index.
     function getDeployerCoordinatorAtIndex(
-        uint256 index
+        uint256 _index
     ) external view returns (address) {
-        return _deployerCoordinators[index];
+        return _deployerCoordinators[_index];
     }
 
     /// @notice Returns the deployer coordinators with an index between the
-    ///         starting and ending indexes (inclusive).
-    /// @param startIndex The starting index (inclusive).
-    /// @param endIndex The ending index (inclusive).
+    ///         starting and ending indexes.
+    /// @param _startIndex The starting index (inclusive).
+    /// @param _endIndex The ending index (exclusive).
     /// @return range The deployer coordinators within the specified range.
     function getDeployerCoordinatorsInRange(
-        uint256 startIndex,
-        uint256 endIndex
+        uint256 _startIndex,
+        uint256 _endIndex
     ) external view returns (address[] memory range) {
         // If the indexes are malformed, revert.
-        if (startIndex > endIndex) {
+        if (_startIndex >= _endIndex) {
             revert IHyperdriveFactory.InvalidIndexes();
         }
-        if (endIndex >= _deployerCoordinators.length) {
+        if (_endIndex > _deployerCoordinators.length) {
             revert IHyperdriveFactory.EndIndexTooLarge();
         }
 
         // Return the range of instances.
-        range = new address[](endIndex - startIndex + 1);
-        for (uint256 i = startIndex; i <= endIndex; i++) {
+        range = new address[](_endIndex - _startIndex);
+        for (uint256 i = _startIndex; i < _endIndex; i++) {
```

Comment:
up to you if it is worth it, but elsewhere we pre-increment an unchecked counter